pkg/restapi/oidc/persistedData.go

```diff
@@ -61,4 +60,13 @@ func (e *persistedData) put(k string, v interface{}) error {
 	return e.s.Put(k, bits)
 }
 
+func (e *persistedData) getEndUser(sub string) (*endUser, error) {
```

Comment:
looks like that the returned `endUser` is not used, maybe we don't want to get the user but, instead, check if he exists - if this is the intention plz update the name of that function to reflect that

Reply:
The general intent is to populate the frontend with the user's name etc.